packages/smooth_app/lib/cards/product_cards/smooth_product_card_not_found.dart

```diff
@@ -19,19 +19,19 @@ class SmoothProductCardNotFound extends StatelessWidget {
       elevation: elevation,
       borderRadius: const BorderRadius.all(Radius.circular(15.0)),
       child: Container(
-        decoration: const BoxDecoration(
-          color: Colors.white,
-          borderRadius: BorderRadius.all(Radius.circular(15.0)),
+        decoration: BoxDecoration(
+          color: Theme.of(context).cardColor,
+          borderRadius: const BorderRadius.all(Radius.circular(15.0)),
         ),
         child: Column(
           mainAxisSize: MainAxisSize.max,
           mainAxisAlignment: MainAxisAlignment.center,
           children: <Widget>[
-            const Text('This product is missing'),
+            Text('This product is missing', style: Theme.of(context).textTheme.bodyText1),
```

Comment:
By curiosity, is there a default textTheme for `Text`, and wouldn't that be `bodyText1`?

Reply:
Just tried and it does not default to `bodyText1` for some reason

Reply:
I checked again. In facts, it is the case that bodyText2 and not bodyText1 is the standard text. Since the color of the BodyText2 has only changed into an accent, no difference was seen in the text without style. I'll add the revised names to the PR right away